src/three/TilesRenderer.js

```diff
@@ -247,6 +262,32 @@ export class TilesRenderer extends TilesRendererBase {
 				} );
 
 			}
+			Promise.resolve().then( () => {
+
+				if ( this.isGeoReferenced === undefined ) {
+
+					this.isGeoReferenced = new Vector3().setFromMatrixPosition( this.root.cached.boxTransform ).length() > 6360000; // approximate Earth radius
+
+				}
+
+				if ( this.isGeoReferenced ) {
+
+					const position = new Vector3().setFromMatrixPosition( this.root.cached.boxTransform );
+					const distanceToEarthCenter = position.length();
+
+					const surfaceDir = position.normalize();
+					const rotationToNorthPole = this.rotationBetweenDirections( surfaceDir, Y_AXIS );
+
+					this.group.quaternion.x = rotationToNorthPole.x;
+					this.group.quaternion.y = rotationToNorthPole.y;
+					this.group.quaternion.z = rotationToNorthPole.z;
+					this.group.quaternion.w = rotationToNorthPole.w;
+
+					this.group.position.y = - distanceToEarthCenter;
+
+				}
+
+			} );
```

Comment:
I don't think repositioning or rotating the tile set based on assumptions about whether a tileset is geopositioned or not should be a part of the core behavior. This should be the responsibility of the application. Perhaps we can move this to the example page, as well?

Reply:
moved

Reply:
Great thanks!